common/views/components/ButtonOutlined/ButtonOutlined.tsx

```diff
@@ -10,15 +10,18 @@ import {
   ButtonIconWrapper,
 } from '../ButtonSolid/ButtonSolid';
 
-type MaybeAnchor = {
+type OutlinedButtonProps = {
   href?: string;
+  isOnDark?: boolean;
 };
 
-export const OutlinedButton = styled(BaseButton).attrs<MaybeAnchor>(props => ({
-  className: classNames({
-    'link-reset': !!props.href,
-  }),
-}))`
+export const OutlinedButton = styled(BaseButton).attrs<OutlinedButtonProps>(
```

Comment:
Why is the component name different to the file name?

Reply:
This isn't the React component that the file exports – it's a styled component for use elsewhere. I have previously wondered what a good strategy for making this kind of distinction clear would be. For example, should any exported styled components follow a simple naming strategy so we know what it is we're dealing with at any given time?

Reply:
If we're exporting a styled component, why not just have them as their own component? How are they different?

Reply:
you mean, at the point we realise we're going to reuse a styled component from one React component, move it into its own component file?

Reply:
Yip.